lib/core/infrastructures/repository/quote_repository.dart

```diff
@@ -13,14 +14,14 @@ class QuoteRepository implements IQuoteRepository {
   final String _baseURL = dotenv.get('API_ENV') != 'production'
       ? dotenv.get('QUOTE_API')
       : dotenv.get('STAGING_QUOTE_API');
-  final String _repositoryURL = '/api/random';
+  final String _repositoryURL = '/api/v1/flirt/quote/random';
```

Comment:
/api suffix in API_BASE_URL and staging

Reply:
Ay sir, sa kani nga comment /api, kay dili sya pwede ma move sa baseUrl kay sa params nga https, ang gina accept niya is tung hostname lang sir, wala labot ang path. Like `staging.sample.com`, dili niya dawatun ang `staging.sample.com/api`.

```dart
Uri.https(
  String authority, [
  String unencodedPath,
  Map<String, dynamic>? queryParameters,
])
```

Reply:
Ah cge2